hack/dump_scenario_from_k8.sh

```diff
@@ -8,15 +8,16 @@ usage() {
     cat <<-EOF
     usage: dump_scenario_from_k8s.sh options
 
-    Dump the current scenario from a running trento-server installation on the k8s cluster
+    Dump the current scenario and discarded discovery events from a running trento-server installation on the k8s cluster
 
     OPTIONS:
         -n, --name            The name to use for the scenario. Defaults to "current".
         -p, --path            The path where the scenario should be saved. Defaults to the current directory.
+        -d, --event-number    Then number of discarded events to dump. Default to 10.
```

Comment:
why this? couldn't we dump just all the discarded events?

Reply:
Considering that this action is for debugging purposes, and that most commonly, the discarded event will be repeated often, I thought that limiting it would be a good idea.

Example:
We have a discarded message sent by the cluster discovery, which by default is sent every 10 seconds. There are possibilities that the 2 nodes are sending an invalid message.

If the landscape has been working just 1 hour. We would already have a file with 720 events.
I don't think we want this hehe More if we start getting dumps with big landscape working days or weeks.

I think having the flexibility to limit the messages is a good feature

Reply:
Alright, I see the rationale and I agree with keeping it, just a couple of points that I would tackle before merging:

-  increase the default 10 to 100 so it's still easy to search for interesting events but we have more of them in case 
- rename event-number to something less generic such as discarded-events-number
- add the discarded events in the pruning job so they get deleted and the table does not grow forever